views/tx.tmpl

```diff
@@ -130,6 +126,7 @@
                                 <div class="hash"><a href="/explorer/address/{{.}}">{{.}}</a></div>
                             {{end}}
                         </td>
+                        <th>{{.ScriptPubKeyDecoded.Type}}</th>
```

Comment:
nice, placement looks good
Although I think non-bold and maybe italics would be good for this data

Reply:
While here, please remove the `{{ if ne .Value 0.0}}` condition since we will want to show all outputs regardless of value